lib/__tests__/slug-utils.test.ts

```
@@ -0,0 +1,367 @@
+
+      // Restore original window
+      global.window = originalWindow
+    })
+
+    it('uses server-side implementation with custom options when window is undefined', () => {
+      // Mock server environment by deleting global.window
+      const originalWindow = global.window
+      // @ts-expect-error - Intentionally deleting window to simulate server
+      delete global.window
+
+      const html = `
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)